Gruntfile.js

```diff
@@ -471,9 +457,8 @@ module.exports = function ( grunt ) {
         src: [
           '<%= vendor_files.compile.js %>',
           '<%= build_dir %>/src/**/*.js',
-          '<%= build_dir %>/assets/vendor/**/*.css',
-          '<%= build_dir %>/assets/vendor/**/*.js',
-          '<%= html2js.common.dest %>',
+          '<%= build_dir %>/assets/node_modules/**/*.css',
```

Comment:
Do we want to include all css and js from node_modules? Can we just include the historic ones that were needed for AngularJS?

Reply:
Actually... do we need `index` any more? Aren't these now all compiled into scripts by angular?